pos_quick_logout/redme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
please remove this whole file. this part will be autogenerated by oca-readme generator tool, when merging. 
it will be dynamic, and avoid errors. (like here, the branch 9.0 is not the good one ;-))